material-components/material-components-ios
# [TextFields] Preserve placeholder font.

When the `placeholderLabel.font` is different from the `font` property on an
MDCTextInputCommonFundament, it was possible that the placeholder font would
be overwritten by the `.font` property accidentally.

|Before|After|
|---|---|
|![textfield-font-broken](https://user-images.githubusercontent.com/1753199/64789910-32019380-d543-11e9-90c0-5362c54cc18c.gif)|![textfield-fontfix-after](https://user-images.githubusercontent.com/1753199/64789916-35951a80-d543-11e9-9a67-e5b15169fc66.gif)|

**Testing Steps**
1. Create an MDCTextField and MDCTextInputControllerFilled
2. Assign different fonts to the controller's `textInputFont` and
   `inlinePlaceholderFont`.
3. Tap into the text field to have it become the first responder and observe
   the placeholder font.
4. Type one letter into the text field and observe the placeholder font.

**Expected Outcome**
Typing the letter does not cause the placeholder font to change.

Closes #8390
